tools/kompos/src/tracingData.ts

```diff
@@ -0,0 +1,110 @@
+/* jshint -W097 */
```

Comment:
For new files, please follow the existing convention, then this would be `tracing-data.ts`.
Or perhaps `history-data.ts`? Think it would be good if it matches the main class to ease navigation. Don't have a particular preference for TracingData or HistoryData, but should match the class.